main.tf

```diff
@@ -23,8 +23,8 @@ locals {
 data "archive_file" "lambda_asg" {
   count       = var.asg_scheduler != null ? 1 : 0
   type        = "zip"
-  source_dir  = "${path.module}/lambda/ec2_asg/update_capacity"
-  output_path = "${path.module}/lambda/ec2_asg/update_capacity/package.zip"
+  source_dir  = "${path.module}/lambda_functions/ec2_asg/update_capacity"
```

Comment:
lambda is a reserved keyword in python 🔴 Renamed to lambda_functions

Reply:
??? we have the "lambda" directory on https://github.com/DND-IT/disco-sitemaps/blame/master/deploy/terraform/lambda.tf and it's not an issue ? Don't get the point

Reply:
yeah we used to use lamda for anything not a python lambda. lambda_functions is more concrete anyhow